relay-server/src/metrics_extraction/event.rs

```diff
@@ -1053,6 +1053,24 @@ mod tests {
                         "ai_total_tokens_used": {
                             "value": 20
                         }
+                    },
+                    "data": {
+                        "ai.pipeline.name": "Autofix Pipeline"
```

Comment:
This is the name of the parent span which represents a specific AI pipeline.

This gets converted into the tag `span.ai.pipeline.group` which can then be queried in the frontend to find how many tokens were used in total for a single span

Span -> span group -> "foreign key join" on ai.pipeline.group -> sum (counter)